pontoon/base/models.py

```diff
@@ -228,6 +240,15 @@ class Project(models.Model):
     # Whether this project has changed since the last sync.
     has_changed = models.BooleanField(default=False)
 
+    # Most recent translation approved or created for this project.
+    latest_translation = models.ForeignKey(
+        'Translation',
+        blank=True,
+        null=True,
+        related_name='+',
```

Comment:
Nah, it's a naming conflict. By default the `related_name` on the Translation for the Locale that it is the `latest_translation` of would be `translation.locale`, but we already have an attribute with that name on the Translation.

And then I copy-pasted it around since, as you said, we'll never use it, but it's only truly necessary on the Locale class.